src/openrct2/actions/GameAction.cpp

```diff
@@ -306,6 +306,7 @@ namespace GameActions
             if (!finance_check_affordability(result->Cost, action->GetFlags()))
             {
                 result->Error = GA_ERROR::INSUFFICIENT_FUNDS;
+                result->ErrorTitle = STR_RIDE_CONSTRUCTION_CANT_CONSTRUCT_THIS_HERE;
```

Comment:
This is not correct. The error title should always come from the game action.

Reply:
Note that the action actually returns `GA_ERROR::OK`, so it wouldn't have any of those anyway.

I think this is just a shortcut to check for funds in a single place, other than in each and every action. Seems like refactoring this out of here is a matter for another PR, don't you agree?

Reply:
Seems correct to me as all game actions have this checked before going into the actual game action, can you elaborate @duncanspumpkin

Reply:
This will mean that all actions that have insufficient funds will have an error title that is ride construction can't construct here

Reply:
I guess thats true but also on the other hand it would duplicate a lot of code if all game actions were to do this. I'm not sure what the optimal way is. I can only think of checking what the game action type is and based on that use a different title maybe?

Reply:
But isn't the real issue that one of the game actions is setting the title incorrectly?

Reply:
When the funds are checked the game action had no Query/Execute call yet, so no game action really assigns a title for the insufficient funds check. Maybe a generic title for this is the best way?

Reply:
Maybe `STR_1161    :Can't position this here...` ?

Reply:
That wouldn't fit when you try to remove something that costs money.

Reply:
Then I guess we're left with `STR_5637    :Can't do this...`

Reply:
Sounds okay, what do you think @duncanspumpkin?

Reply:
I still think there is something better we could do but that will be sufficient for now

Reply:
Ok